ocfweb/templates/base.html

```diff
@@ -207,7 +209,12 @@
                         <p>The Open Computing Facility is run entirely by student volunteers.</p>
                         <p>Copyright &copy; 1989&ndash;{% now 'Y' %} Board of Directors of the Open Computing Facility.</p>
                         <p>The Open Computing Facility is a Chartered Program of the ASUC.</p>
-                    </div>
+                        <p>All our software including this website is open source.</p>
```

Comment:
small nitpick but I think grammatically it should be "All of our software, including this website, is open source"